coding-concepts.js

```diff
@@ -7,49 +7,49 @@
 // There is no need to change any of the code.
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
-const colors = ["tangerine", "magenta", "lilac", "daffodil"]
+// const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
 // a) Your answer:
 // b) Verify and explain:
 
 // --------------------1) What will this log?
 
-const cohort = "LEARN 2023"
+// const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: 10
+// // b) Verify and explain: Correct because the string has 10 letters and length isn't checking for the number of indexes.
 
-// --------------------2) What will this log?
+// // --------------------2) What will this log?
 
-const greeting = "Hello World!"
+// const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer:4
+// // b) Verify and explain:Correct b/c the 4 passed in the square brackets indicates to print the value in the 4th index.
```

Comment:
your explanation is correct, however you answer is a bit off. Because the [4] is indicating that we want the value at the 4th index the answer would be "o"